### ISSUE TITLE
training scripts

### ISSUE THREAD

**Issue by GuoleiSun** [gasvn/Res2Net#1]
Hi,

Thanks for your work. Can you share the training scripts to reproduce the those numbers (top-1 and top-5 accuracy)?

Thanks a lot

**Comment by gasvn** [comment]
Sorry, due to the regulation of our lab, we can only release the trained models for academic use.
If you want to acquire the training scripts for business use, please contact the leader of our lab.

I follows the common training strategy to train our model. You can refer to this training script if you want to modify the Res2Net with your own idea: https://github.com/pytorch/examples/blob/master/imagenet/main.py

Also, feel free to contact me with shgao@live.com in case I miss the issues.
